tests/interfaces/records/handlers/records-query.spec.ts

```diff
@@ -98,6 +99,38 @@ describe('RecordsQueryHandler.handle()', () => {
       expect(reply2.entries?.length).to.equal(1); // only 1 entry should match the query
     });
 
+    it('should return `encodedData` if data size is within the spec threshold', async () => {
+      const data = TestDataGenerator.randomBytes(DwnConstant.maxDataSizeAllowedToBeEncoded); // within/on threshold
+      const alice = await DidKeyResolver.generate();
+      const write= await TestDataGenerator.generateRecordsWrite({ requester: alice, data });
+
+      const writeReply = await dwn.processMessage(alice.did, write.message, write.dataStream);
+      expect(writeReply.status.code).to.equal(202);
+
+      const messageData = await TestDataGenerator.generateRecordsQuery({ requester: alice, filter: { recordId: write.message.recordId } });
+      const reply = await dwn.processMessage(alice.did, messageData.message);
+
+      expect(reply.status.code).to.equal(200);
+      expect(reply.entries?.length).to.equal(1);
+      expect((reply.entries[0] as any).encodedData).to.equal(Encoder.bytesToBase64Url(data));
```

Comment:
why `as any`? My typescript isn't super deep, but I would assume that it can pick up that `entries` is `QueryResultEntry[]`, which have property `encodedData`.

Reply:
Thanks for bringing this up! This was a copy and past from existing tests before I decided to refactor the code. It was needed before the the intro of `QueryResultEntry`, but no longer needed now.

Went ahead and removed all the `as any` in tests.

Reply:
Thanks for the extra clean up!
Not to punish a good deed, but I see a few more in records-query.spec.ts that could be cleaned up. Found them in VSCode CMD-F with regex `entries\[.\] as any\).encodedData`. Do you mind cleaning up those too?

Reply:
I am not seeing any more, can you point me to an example?

Reply:
Ope, sorry I think my VSCode was showing me the wrong commit. Apologies, still getting used to this editor.